app/client/src/pages/Editor/IDE/EditorTabs/SplitScreenTabs.tsx

```diff
@@ -60,12 +74,23 @@ const SplitScreenTabs = () => {
         <Container>
           <ToggleButton
             icon="add-line"
+            id="tabs-add-toggle"
             isSelected={segmentMode === EditorEntityTabState.Add}
             onClick={onAddClick}
             size="md"
           />
           <FileTabs navigateToTab={onClick} tabs={files} />
           <ListButton items={overflowList} navigateToTab={onClick} />
+          <Tooltip content={createMessage(MAXIMIZE_BUTTON_TOOLTIP)}>
+            <Button
+              className="ml-auto"
```

Comment:
add data-testId here
```suggestion
              className="ml-auto"
              data-testid="t--ide-maximize"
```